src/rgw/rgw_main.h

```diff
@@ -63,12 +63,12 @@ class AppMain {
   std::vector<RGWFrontend*> fes;
   std::vector<RGWFrontendConfig*> fe_configs;
   std::multimap<string, RGWFrontendConfig*> fe_map;
-  std::unique_ptr<rgw::LDAPHelper> ldh;
```

Comment:
I don't know that I prefer this--namespace qualifiers help developers learn where things are